barretenberg/cpp/src/barretenberg/goblin/CMakeLists.txt

```diff
@@ -1 +1 @@
-barretenberg_module(goblin stdlib_recursion ultra_honk eccvm translator_vm)
\ No newline at end of file
+barretenberg_module(goblin stdlib_recursion ultra_honk eccvm translator_vm stdlib_sha256 stdlib_merkle_tree stdlib_primitives)
```

Comment:
The inclusion of the stdlib components is only necessary now because the full goblin recursion tests (which now use these stdlib primitives in the kernel) live in the goblin module. We don't have an established pattern for avoiding this because this is the only place that we use stdlib primitives outside the stdlib (except for certain bench suites). One option is just to have a goblin_test module that is separate from goblin. Let's discuss

Reply:
If they're only in headers, this isn't technically needed. Also, might signal that this is a break vs having them in a testing module